VideoPlayer follow-up (#655)
primer/brand: apps/docs/content/components/VideoPlayer.mdx

</VideoPlayer>
```

## Custom play icon

```jsx live
<VideoPlayer
  title="GitHub media player"
  playIcon={() => <PlayIcon size={96} />}
>
  <VideoPlayer.Source
    src="https://primer.github.io/brand/assets/example.mp4"
    type="video/mp4"
  />
  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
</VideoPlayer>
```

## Controlling programmatically

> The `VideoPlayer` component exposes a `useVideo` hook that can be used to control the video programmatically. To use the hook, the `VideoPlayer` component must be wrapped in a `VideoPlayer.Provider` component.

Reviewer: Minor clarification for accuracy..

```suggestion
A `useVideo` hook is available and can be used to control the video programmatically. To use the hook, the `VideoPlayer` component must be wrapped in a `VideoPlayer.Provider` component.
```

```tsx live
() => {
  const MyVideoPlayer = () => {
    const {isPlaying, togglePlaying, seek} = useVideo()

    return (
      <Stack direction="vertical">
        <VideoPlayer
          title="GitHub media player"
          showPlayPauseButton={false}
          showSeekControl={false}
          showCCButton={false}
          showMuteButton={false}
          showVolumeControl={false}
          showFullScreenButton={false}
        >
          <VideoPlayer.Source
            src="https://primer.github.io/brand/assets/example.mp4"
            type="video/mp4"
          />
          <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
        </VideoPlayer>
        <Stack direction="horizontal">
          <Button onClick={() => togglePlaying()}>
            {isPlaying ? 'Pause' : 'Play'}
          </Button>
          <Button onClick={() => seek(0)}>Go to start</Button>
          <Button onClick={() => seek((t) => t + 5)}>Skip 5 seconds</Button>
        </Stack>
      </Stack>
    )
  }

  return (
    <VideoPlayer.Provider>
      <MyVideoPlayer />
    </VideoPlayer.Provider>
  )
}
```

> <h2>useVideo Context</h2>

Reviewer: Can we down level this to h3 and rename for clarity. We tend to prefer markdown headings unless you need to add additional attributes in there. Just a stylistic point...

```suggestion
### useVideo hook
```

Author: Not sure how I managed to start using HTML over markdown there, good catch, thanks

| `kind` | `'subtitles'`, `'captions'`, `'descriptions'`, `'chapters'`, `'metadata'` | `'captions'` | `false`  | Sets how the text track is meant to be used |

The component API supports all standard HTML attribute props, while providing some additional behavior as described above.

<h3>
  VideoPlayer.Provider
</h3>

> `VideoPlayer.Provider` is used in conjunction with the `useVideo` hook to provide a comprehensive API for managing video playback, volume, closed captioning, and fullscreen mode.

Reviewer: ```suggestion
`VideoPlayer.Provider` can be used in conjunction with the `useVideo` hook to enable programmatic access to features such as video playback, volume, closed captioning, and fullscreen mode.
```

Author: 👍
